kubeflow-katib.yaml

```diff
@@ -67,165 +67,165 @@ subpackages:
     description: Kubeflow Katib earlystopping
     dependencies:
       runtime:
-        - py3-grpcio
-        - py3-protobuf
-        - py3-googleapis-common-protos
-        - py3-kubernetes
-        - cython
+        - python-3.11-dev
+        - py3.11-pip
     pipeline:
       - runs: |
+          python -m venv .venv-earlystopping --system-site-packages
           export SUGGESTION_DIR="cmd/earlystopping/medianstop/v1beta1"
+          .venv-earlystopping/bin/pip install -r $SUGGESTION_DIR/requirements.txt
           mkdir -p ${{targets.subpkgdir}}/opt/katib
           mkdir -p ${{targets.subpkgdir}}/opt/katib/pkg
           mkdir -p ${{targets.subpkgdir}}/opt/katib/$SUGGESTION_DIR
           tar c ./pkg | tar xvf - -C ${{targets.subpkgdir}}/opt/katib/
+          mv .venv-earlystopping/* ${{targets.subpkgdir}}/opt/katib/$SUGGESTION_DIR
           mv ./$SUGGESTION_DIR/* ${{targets.subpkgdir}}/opt/katib/$SUGGESTION_DIR
-
+  
   - name: katib-tfevent-metricscollector
     description: Kubeflow Katib tfevent-metricscollector
+    dependencies:
+      runtime:
+        - openmp-17
+        - python-3.11-dev
+        - py3.11-pip
```

Comment:
according to the error in CI, we should add `openmp-17` here

Reply:
Done